cluster/images/etcd/cloudbuild.yaml

```diff
@@ -14,6 +14,7 @@ steps:
       - IMAGE=gcr.io/$PROJECT_ID/etcd
       - BUILD_IMAGE=debian-build
       - TMPDIR=/workspace
+      - HOME=/root  # for docker buildx
```

Comment:
OK, I see this used elsewhere for the same reason: https://cs.k8s.io/?q=HOME%3D%2Froot&i=nope&files=&excludeFiles=&repos=